website/docs/getting-started/setup/instance-management/update-appsmith.md

```diff
@@ -0,0 +1,53 @@
+---
+description: Follow the steps to update Appsmith to the latest version.
+---
+
+# Update Appsmith
+This page provides instructions to update Appsmith to the latest version.
+
+## Prerequisites
+Ensure that you complete the following steps before starting the update process.
+
+* For backup and update tasks, you need at least 2 GB of free storage.
+* You need to create a backup of the Appsmith instance before a manual update. To create a backup, see the [Backup Appsmith Instance](/getting-started/setup/instance-management/appsmithctl#backup-appsmith-instance) section.
```

Comment:
```suggestion
* You need to create a backup of the Appsmith instance before a manual update. To create a backup, see the [Backup Appsmith Instance](/getting-started/setup/instance-management/appsmithctl#backup-appsmith-instance) section.
```
* It is highly recommended to create a backup of the Appsmith instance before a manual update. To create a backup, see the [Backup Appsmith Instance](/getting-started/setup/instance-management/appsmithctl#backup-appsmith-instance) section.